apps/api/src/auth/services/user-auth-token/user-auth-token.service.ts

```diff
@@ -1,14 +1,20 @@
 import { singleton } from "tsyringe";
 
 import { kvStore } from "@src/middlewares/userMiddleware";
-import { env } from "@src/utils/env";
 import { getJwks, useKVStore, verify, VerifyRsaJwtEnv } from "@src/verify-rsa-jwt-cloudflare-worker-main";
+import { AuthConfigService } from "../auth-config/auth-config.service";
 
 @singleton()
 export class UserAuthTokenService {
+  readonly #authConfigService: AuthConfigService;
+
+  constructor(authConfigService: AuthConfigService) {
```

Comment:
why is this like so instead of regular
```suggestion
  constructor(private readonly authConfigService: AuthConfigService) {
```

is this to use # notation? I assume DI works this way too?

Reply:
hashbang private fields do not work with short notation in constructor.